queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals WHERE name = 'mon';
+SELECT * animals WHERE date_of_birth = '2016 - 2019';
+SELECT * animals WHERE escape_attempts > 2 AND neutered = true;
```

Comment:
- [ ] Kindly for the first 3 queries they are not returning the right data, Kindly recheck them and be sure that they returning the right data.

Reply:
this has been implemented thank you